src/Core/Session.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace TrendingRepos\Core;
+
+class Session {
+    public function __construct()
+    {
+        if($this->isStarted())
+        { 
+            session_start(); 
+        } 
+    }
+
+    public function isStarted(): bool
+    {
+        return session_status() === PHP_SESSION_ACTIVE;
+    }
+
+    public function set(string $key, string $value)
+    {
+        $_SESSION[$key] = $value;
+    }
+
+    public function get(string $key) 
```

Comment:
```suggestion
    public function get(string $key): ?string
```

Reply:
So we add `?` if the return type might be `null` not a `string` right ?